packages/hardhat/deploy/00_deploy_your_contract.ts

```diff
@@ -31,7 +32,7 @@ const deployYourContract: DeployFunction = async function (hre: HardhatRuntimeEn
     autoMine: true,
   });
 
-  // const yourContract = await hre.ethers.getContract("YourContract", deployer);
+  // await hre.ethers.getContract<Contract>("YourContract", deployer);
```

Comment:
Maybe change `YourContract` with `BatchRegistry`? So the example works out of the box when uncommenting.

Reply:
Ah yeah, good call :+1: